docs/supervisor/development.md

```diff
@@ -23,6 +23,25 @@ When the initializing is complete you can access the Home Assistant frontend on
 
 If you need to rebuild the base of the Supervisor run the task "Build Supervisor" and then restart the "Run Supervisor" task again when that completes.
 
+#### Using Windows WSL2
+While the system is Linux and compatible with Mac, it is also possible to build and test locally on Windows using WSL2 with Ubuntu 20.04 LTS and other similar distros. This is achieved via WSL2, which makes POSIX tasks possible within Windows via system-level integration with a Linux virtual machine. 
```

Comment:
There are some known issues with Ubuntu and Supervisor based installations. I wonder why to use Ubuntu in this case, while Debian is available as well. Is there a specific reason for that?

```suggestion
While the system is Linux and compatible with Mac, it is also possible to build and test locally on Windows using WSL2 with Ubuntu 20.04 LTS and other similar distros. This is achieved via WSL2, which makes POSIX tasks possible within Windows via system-level integration with a Linux virtual machine.
```

Reply:
Unresolving, as there was an additional question in this comment.

Reply:
Was not aware of that.  I always choose Ubuntu for WSL as they are a user-friendly build and were the first by months to release a WSL package.  

Let's draft this for a bit.  I'll work on some other base.

Ubuntu is Debian-based.  Is there another option besides debian?